homeworks/week02/README.md

```diff
@@ -0,0 +1,232 @@
+ will control what kind of HTML you should generate, but you can ignore
+ it for now.
+
+## Step 1:
+
+Begin by creating a function that implements the following signature:
+```javascript
+function renderMovie(movie)
+```
+
+The ```movie``` argument will be a JavaScript object with five fields,
+like this example:
+
+```javascript
+{
+	name: "Minions",
+	year: 2015,
+	genre: "animation",
+	cast: ["Sandra Bullock", "Jon Hamm", "Michael Keaton"],
+	description: "Evolving from single-celled yellow organisms at the dawn of time, Minions live to serve, but find themselves working for a continual series of unsuccessful masters, from T. Rex to Napoleon. Without a master to grovel for, the Minions fall into a deep depression. But one minion, Kevin, has a plan."
+}
+```
+
+Using the attributes of this object, your function should create a new
+string variable, use string concatenation to generate a string of
+HTML, and return that variable with the ```return``` command.
+
+For example, given the above object, you should generate the following
+HTML:
+
+```html
+<div class="movie animation">
+  <h1 class="name">Minions</h1>
+  <p>Release date: <span class="year">2015</span>, Genre: <span class="genre">animation</span>.</p>
+
+  <div>
+    Starring:
+    <ul>
+      <li>Sandra Bullock</li>
+      <li>Jon Hamm</li>
+      <li>Michael Keaton</li>
+    </ul>
+  </div>
+  <p class="description">
+    Evolving from single-celled yellow organisms at the dawn of time,
+    Minions live to serve, but find themselves working for a continual
+    series of unsuccessful masters, from T. Rex to Napoleon. Without a
+    master to grovel for, the Minions fall into a deep depression. But
+    one minion, Kevin, has a plan.
+  </p>
+</div>
+```
+
+For example, to get you started, you might begin with something like
+the following:
+```javascript
+  var returnValue = "";
+
+  returnValue += "<div class=\"movie " + movie.genre + "\">";
+```
+
+Note that ```movie.genre``` will be used twice: as a ```class``` in
+the outer ```<div>``` tag, and also in a ```<span>``` to display to
+the user. Note also that you will need to loop over the array in the
+```movie.cast``` attribute. We suggest using ```forEach```.
+
+Test your function by modifying ```render()``` so that it calls your
+function, passes in the argument ```movieArray[0]```, and returns
+the string that your function generates, like so:
+
+```javascript
+  return renderMovie( movieArray[0] );
+```
+
+Now open ```css/style.css``` and modify it to add some styles that
+make your HTML display a little nicer in the browser. We've provided
+some basic CSS to get you started. At a bare minimum, use the genre
+selectors that we have provided so that each movie is displayed a
+little differently depending on its genre. We would suggest also
+adding some ```margin``` and ```padding``` properties to the
+```.movie``` and ```.description``` selectors as well.
```

Comment:
A background image or icon per genre would be cool? Maybe describe the `content:url` CSS attribute here?